README.md

```diff
@@ -1,3 +1,22 @@
-## NextJS template
+<p align="center">
+  <img width="120" height="120" src="/public/og_image_red.png">
+</p>
+
+## PR-Codex
+
+A github app to summarize code diffs in pull requests, right in the PR discussion on Github. Powered by ChatGPT.
+
+## Get started
+
+1. [Install the app on your repositories](https://github.com/apps/pr-codex/installations/new)
+2. Open a PR or push changes to an existing one, and PR-Codex will summarize the code diffs in a comment.
+
+![Demo PR-Codex message](/public/demo.jpg)
+
+## Pricing
+
+PR-Codex is currently free.
+
+---
```

Comment:
/ask-codex summarize this

Reply:
/ask-codex summarize this

Reply:
/ask-codex rewrite this markdown content so that it is shorter. keep all links. Use markdown formatting.